app.py

```diff
@@ -216,6 +216,20 @@ def update_link(id_):
     return jsonify(link.serialize())
 
 
+@app.route('/grades/<code_>', methods=['GET'])
```

Comment:
@egiurleo the original issue says this should be named /scores/{code}, but this seemed a bit more fitting. Let me know if we want to change this back!

Reply:
Yes this makes more sense! Thank you!